site/en/docs/web-platform/chrome-variations/index.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: 'layouts/doc-post.njk'
+title: What are Chrome Variations?
+subhead: A mechanism for Chrome to test new browser features
+description: A mechanism for Chrome to test new browser features
+date: 2022-12-20
+authors:
+- samdutton
+---
+
+{% YouTube
+id='teSgavrweWI' 
+%}
+
+You might have heard of Chrome Variations, or Chrome Field Trials—or maybe even the internal
+codename, Chrome Finch.
+
+These are all names for the same thing: a mechanism to test new features or changes to the Chrome
+browser, or to Chrome OS, the Chrome operating system that runs on Chromebooks.
+
+## What are Chrome Variations used for?
+
+Chrome Variations enable Chrome to **activate** a new feature, **deactivate** a feature, or try out
+a **change to a feature** for a **subset of users**. 
+
+Not everything in Chrome is introduced via the Chrome Variations mechanism. However, the Chrome team
+can use Chrome Variations whenever we need to be especially careful about making a change, or where
+there's a risk that a change might affect performance in ways we didn't expect. A 'change' might
+mean a performance enhancement in Chrome's code, an update to how the browser looks or functions, or
+occasionally even a modification to a JavaScript API. 
+
+Chrome Variations can also be used to **validate a hypothesis** about a change or an update.   
+For example, for a percentage of users in a Chrome Variations group, we tried tweaking [QUIC
+networking protocol](https://blog.cloudflare.com/the-road-to-quic/) parameters to make Chrome faster
+for users' real network conditions. We've also conducted Omnibox studies around how many results to
+show for optimal usability. 
+
+Another area where you may encounter Chrome Variations is if you work with Chrome's [origin
+trials](/docs/web-platform/origin-trials/). By default, an **origin trial** feature is enabled on
```

Comment:
```suggestion
trials](/docs/web-platform/origin-trials/). By default, an origin trial feature is enabled on
```

Reply:
> @samdutton are you planning to release these in the main docs architecture? Here: https://deploy-preview-4674--developer-chrome-com.netlify.app/docs/web-platform/

@heyawhite Yes, though I wonder if origin trials should have their own section. (It might be nice to be able to link to all origin trials documentation in one place.)
 
> If yes, the "Chrome Concepts" section could easily be first on the list.

SG!